File: programjava/MaxMin.java
package programjava;
import java.util.Scanner;

public class MaxMin {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.print("Enter first number >");
        int first = scanner.nextInt();
        System.out.print("Enter second number >");
        int second = scanner.nextInt();
        scanner.close();
        if(first > second){
            System.out.println("Maximum Number "+ first);
            System.out.println("Minumum number "+ second);
        }else{
            System.out.println("Maximum Number "+ second);
            System.out.println("Minumum number "+ first);
        }
     }
}

File: c/MaxMin.c
#include<stdio.h>
int main(){
    int first,second;
    printf("Enter first number >");
    scanf("%d",&first);
    printf("Enter second number >");
    scanf("%d",&second);
    
    if(first > second){
        printf("Maximum Number = %d\n",first);
        printf("Minumum number = %d",second);
    }else{
        printf("Maximum Number = %d\n",second);
        printf("Minumum number = %d",first);
    }
    return 0;
}